Makefile

```diff
@@ -1,6 +1,7 @@
 OADP_TEST_NAMESPACE ?= openshift-adp
 REGION ?= us-east-1
 PROVIDER ?= aws
```

Comment:
@deepakraj1997 @hhpatel14 @dymurray may want to revisit renaming provider to something more BSL/BL/ObjectStorage specific